lib/checks/canFindRoots.js

```diff
@@ -0,0 +1,33 @@
+const Node = require('../node');
+const resolvesToConstant = require('./resolvesToConstant.js');
+
+function canFindRoots(equation) {
+
+  // Equation should be in the form of factor * factor = 0, factor^power = 0, or identifier = 0
+  // e.g (x - 2)^2 = 0, x(x + 2)(x - 2) = 0, x = 0
+  const left = equation.leftNode;
+  const right = equation.rightNode;
+
+  const zeroRightSide = Node.Type.isConstant(right)
```

Comment:
it'd be nice to have a helper function where you can just do `Node.Type.isConstant(right, 0)` but I guess you don't need that for now (would be really nice though ^^)